parsers/mongodb/mongodb.go

```diff
@@ -28,34 +38,75 @@ type MongoLineParser struct {
 }
 
 func (m *MongoLineParser) ParseLogLine(line string) (map[string]interface{}, error) {
-	return parser.ParseLogLine(line)
+	return logparser.ParseLogLine(line)
 }
 
-func (p *Parser) Init(_ interface{}) error {
+func (p *Parser) Init(options interface{}) error {
+	p.conf = *options.(*Options)
 	p.nower = &RealNower{}
 	p.lineParser = &MongoLineParser{}
 	return nil
 }
 
+func (p *Parser) parseTimestamp(values map[string]interface{}) (time.Time, error) {
+	now := p.nower.Now()
+	timestamp_value, ok := values["timestamp"].(string)
```

Comment:
maybe pull "timestamp" out in to a constant "timestampFieldName" for typo-protection and magic-identification?